features/README.md

```diff
@@ -35,3 +42,24 @@ De Gherkin scenario's worden opgenomen in de map /features. Voor elke functional
 Deze manier van specificeren maakt het mogelijk om te worden uitgevoerd als geautomatiseerde test van de API. Met deze opzet van de specificatie/test wordt [Behaviour Driven Development (BDD)](https://docs.cucumber.io/bdd) ondersteund.
 
 De features zijn op dit moment niet volledig uitvoerbaar als BDD test. Voor de [referentie-implementatie](https://github.com/lostlemon/haal-centraal-brp-bevragen) is een belangrijk deel van de features in een werkende test omgezet, maar de hier gepubliceerde features zijn dat niet.
+
+## Uitvoeren van scenario's in feature files als end to end tests
```

Comment:
moeten we hier opnemen hoe wij de tests uitvoeren? Anderen hebben immers geen toegang tot de databasedefinitie die nodig is voor de e2e test. zie eerdere opmerking bij docker-compose-e2e-readme